client/scss/components/_page_description.scss

```diff
@@ -0,0 +1,66 @@
+//* @define page-description;
+
+.page-description {
+  &.page-description--wb {
```

Comment:
So here this is Wellcome Bold - but nowhere in the styling is that applied.
There is also a lot of extra styling that has nothing to do with the font.